metagraph/src/annotation/taxonomy/tax_classifier.cpp

```diff
@@ -0,0 +1,205 @@
+#include "tax_classifier.hpp"
+
+#include <cmath>
+#include <string>
+#include <vector>
+
+#include "annotation/representation/annotation_matrix/annotation_matrix.hpp"
+#include "common/unix_tools.hpp"
+
+#include "common/logger.hpp"
+
+namespace mtg {
+namespace annot {
+
+using mtg::common::logger;
+
+TaxonomyClsAnno::TaxonomyClsAnno(const graph::AnnotatedDBG &anno,
+                                 const double lca_coverage_rate,
+                                 const double kmers_discovery_rate,
+                                 const std::string &tax_tree_filepath,
+                                 const std::string &label_taxid_map_filepath) :
+                                 TaxonomyBase(lca_coverage_rate, kmers_discovery_rate), _anno_matrix(&anno) {
```

Comment:
Break lines to make it easier to read and see logical blocks rather than look nice and aligned.
```suggestion
                                 const std::string &label_taxid_map_filepath)
      : TaxonomyBase(lca_coverage_rate, kmers_discovery_rate),
        _anno_matrix(&anno) {
```